packages/react-composites/src/composites/MeetingComposite/ChatButton.tsx

```diff
@@ -14,6 +14,7 @@ export const ChatButton = (props: ControlBarButtonProps): JSX.Element => {
   return (
     <ControlBarButton
       {...props}
+      labelKey={'chatButtonLabelKey'}
```

Comment:
Quickly fixed this also:
![image](https://user-images.githubusercontent.com/2684369/133479215-47028ecb-baf1-40ad-baaf-f842003a0726.png)